src/Numerics/Mesh/Grids.jl

```diff
@@ -697,14 +883,10 @@ function computegeometry(
 
     elseif dim == 3
         map!(
-            JcV,
-            J,
-            ξ1x1,
-            ξ1x2,
-            ξ1x3,
-            ξ2x1,
-            ξ2x2,
-            ξ2x3,
+            #! format: off
+            JcV, J,
+            ξ1x1, ξ1x2, ξ1x3, ξ2x1, ξ2x2, ξ2x3,
+            #! format: on
         ) do J, ξ1x1, ξ1x2, ξ1x3, ξ2x1, ξ2x2, ξ2x3
             x1ξ3 = J * (ξ1x2 * ξ2x3 - ξ2x2 * ξ1x3)
             x2ξ3 = J * (ξ1x3 * ξ2x1 - ξ2x3 * ξ1x1)
```

Comment:
What is this map! (...) do ...  ?  Why is there JcV in map!(...) but not in do(...) ?

Reply:
It's Julia's [do-block](https://docs.julialang.org/en/v1/manual/functions/#Do-Block-Syntax-for-Function-Arguments) syntax

Reply:
`JcV` is the destination of the call, see the docs for [`map!`](https://docs.julialang.org/en/v1/base/collections/#Base.map!)